PostController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+
+    public function postAction() {
+
+        //avoid notices warnings
+        !isset($_POST['livechat_license_number']) ? $livechat_license_number =  '': $livechat_license_number = $_POST['livechat_license_number'];
+        !isset($_POST['livechat_groups']) ? $livechat_groups =  '0' :  $livechat_groups =  $_POST['livechat_groups'];
+        !isset($_POST['livechat_params']) ? $livechat_params =  '' :  $livechat_params =  $_POST['livechat_params'];
+
+        $config_table = Mage::getSingleton('core/resource')->getTableName('core_config_data');
+
+        $read = Mage::getSingleton('core/resource')->getConnection('core_read');
+        $query = 'SELECT * FROM ' . $config_table;
+        $query .= 'WHERE scope="default" AND scope_id=0 AND path="livechat/general/license"';
+        $results = $read->fetchAll($query);
```

Comment:
Not recommend to use direct sql queries to retrieve the db records. Use Models or Resource models to retrieve the records from db table. 
Here we're getting the records from core_config_data table , which means we'll retrieve the system configuration value for "livechat/general/license" field. Using M1 we can retrieve the value using 
Mage::getStoreConfig('livechat/general/license');